### PR TITLE
Tommy bookings

### PR DESCRIPTION
**Pull request by Karituo-Luster**
Added documentation for Bookings object & 4 BookingService
- findAllBookingsByMemberId
- calculateTotalPrice
- calculateSeatPrice
- calculateLuggagePrice

### CODE REVIEW

File: src/main/java/com/revature/ams/Booking/BookingService.java [hunk 1]
```diff
@@ -45,7 +45,9 @@ public List<Booking> findAll() {
     }
 
     /**
-     * TODO: DOCUMENT ME
+     * This service method retrieves all bookings made by a specific member based on their memberId.
+     * The sends data generated from the bookingRepository and sends it to the bookingController.
```
Comment (Gilberto787) [inline src/main/java/com/revature/ams/Booking/BookingService.java:49]: Small typo should be Then rather then The

**Review by Gilberto787 — COMMENTED** [review]
looks good

File: src/main/java/com/revature/ams/Booking/BookingService.java [hunk 2]
```diff
@@ -83,7 +88,12 @@ private BigDecimal calculateSeatPrice(Booking.SeatType seatType) {
     }
 
     /**
-     * TODO: DOCUMENT ME
+     * The method calculates the total price of luggage based on the number of checked luggage, including discounts.
+     * If the number of checked luggage is less than 0, the method throws an IllegalArgumentException.
```
Comment (CAngeley) [inline src/main/java/com/revature/ams/Booking/BookingService.java:92]: Should use @ throws to make notice of the potential exception.

File: src/main/java/com/revature/ams/Booking/BookingService.java [hunk 3]
```diff
@@ -68,7 +72,8 @@ public BigDecimal calculateTotalPrice(Booking booking) {
     }
 
     /**
-     * TODO: DOCUMENT ME
+     * The method assigns the seat price based on the seat type.
+     * Throws an IllegalArgumentException if the seat type is invalid.
```
Comment (CAngeley) [inline src/main/java/com/revature/ams/Booking/BookingService.java:76]: Should use @ throws to make notice of the potential exception.

File: src/main/java/com/revature/ams/Booking/Booking.java [hunk 4]
```diff
@@ -33,7 +33,9 @@ public Booking(int bookingId, Flight flight, Member member, boolean carryOnAllow
     }
 
     /**
-     * TODO: DOCUMENT ME
+     * This is a constructor that takes a BookingRequestDTO and maps it to a Booking object.
+     * The booking object contains information about the flight number, the member id, 
+     * number of luggage checked in, and the seat type.
      *
      * @param bookingRequestDTO
```
Comment (Luis-Cruz1027) [inline src/main/java/com/revature/ams/Booking/Booking.java:40]: I think here you should definitely give more insight into what the parameter is

File: src/main/java/com/revature/ams/Booking/BookingService.java (hunk 1 above)
Comment (Karituo-Luster) [reply src/main/java/com/revature/ams/Booking/BookingService.java:49]: noooo-emote

**Pull request merged**